tests/admin/test_api.py

```diff
@@ -218,9 +218,13 @@ def test_predefined_system_queries(admin_api: FlaskClient) -> None:
     assert data[0]["name"] == "CurrentMerges"
 
 
+@pytest.mark.xfail
 @pytest.mark.redis_db
 @pytest.mark.clickhouse_db
 def test_query_trace(admin_api: FlaskClient) -> None:
+    """
+    TODO-23.3: Looks like the tracing output has changed:
```

Comment:
What was the problem with this? I can try to fix it.

Reply:
@evanh The test assumes that the `formatted_trace_output` has data in the `read_performance` but from the payload of the test is the following

`{'5b9af86924cb': {'aggregation_performance': ['Aggregator: Merging partially aggregated single-level data.', 'Aggregator: Converted aggregated data to blocks. 1 rows, 8.00 B in 3.167e-06 sec. (315756.236 rows/sec., 2.41 MiB/sec.)'], 'memory_performance': ['MemoryTracker: Peak memory usage (for query): 225.13 KiB.'], 'node_name': '5b9af86924cb', 'node_type': 'query', 'number_of_storage_nodes_accessed': 0, 'read_performance': [], 'storage_nodes_accessed': [], 'thread_ids': ['399', '390'], 'threads_used': 2}}`

specifically `'read_performance': [],`

Reply:
https://github.com/getsentry/snuba/pull/4939